config/hardware/probes/tap_probe.cfg

```diff
@@ -1,6 +1,6 @@
 [gcode_macro _USER_VARIABLES]
 variable_probe_type_enabled: "vorontap"
-variable_startprint_actions: "bed_soak", "chamber_soak", "clean", "extruder_preheating", "tilt_calib", "z_offset", "bedmesh", "extruder_heating", "purge", "clean", "primeline"
+variable_startprint_actions: "extruder_preheating", "bed_soak", "chamber_soak", "clean", "tilt_calib", "z_offset", "bedmesh", "extruder_heating", "purge", "clean", "primeline"
```

Comment:
i suggest to do the extruder preheating after bed_soak, as the bed soak takes the longest amount of time. no need to waste power on the extruder heating while waiting for bed soak

Reply:
Mhmm I'm open to move it where you prefer :)

  - First position allow to heat the chamber quickly and save time as we do it in parallel to the bed soak but this is not always needed
  - After bed soak (but before chamber soak) make it more generic if no chamber soak is needed and allow to save some energy but can make the process a little bit longer if a chamber soak is needed

So ok, let's go for more genericity by putting it in second position. Users will be able override it if they want to put it in front anyway.